python-sdk/nuscenes/eval/lidarseg/validate_submission.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import json
+import os
+
+import numpy as np
+from tqdm import tqdm
+
+from nuscenes import NuScenes
+from nuscenes.eval.lidarseg.utils import get_samples_in_eval_set
+from nuscenes.utils.data_classes import LidarPointCloud
+
+
+def validate_submission(nusc: NuScenes, results_folder: str, eval_set: str, verbose: bool = False) -> None:
```

Comment:
is this highlighted in the README?

Reply:
Have added in a sentence to indicate this